src/components/browse/datasetlist/datasetListHeader.tsx

```diff
@@ -17,22 +17,31 @@ const DatasetListHeader = (props: dataSetListHeaderProps) => {
     ) {
       let searchParamsList = props.searchParams.get("f").split(";");
       for (var item of searchParamsList) {
-        filterParamsList.push(item.replace(":", " | "));
+        filterParamsList.push(item.replace(":", ": "));
```

Comment:
why is this changed?

Reply:
It took slightly less space.

Reply:
This is a cosmetic feature, I think a "pipe" is lot of visible than the "colon". We can check with the design team if they have any preference and update it.